src/handlers/v1/markets/[chainId]/[marketId]/deleverage/index.ts

```diff
@@ -79,8 +79,8 @@ export const v1GetDeleverageQuotationRoute: Route<GetDeleverageQuotationRoutePar
 
       // 1. get the quotation for swapping the deleverage token into amount of the base token.
       const quotation = await apisdk.protocols.paraswapv5.getSwapTokenQuotation(chainId, {
-        tokenIn: deleverageCollateralToken,
-        output: { token: baseToken, amount },
+        tokenIn: deleverageCollateralToken.wrapped,
+        output: { token: baseToken.wrapped, amount },
```

Comment:
Should we wrap `deleverageCollateralToken` on line 88 although it won't affect `gt`? Or simply compare `quotation.input.amountWei` with `new BigNumberJS(deleverageCollateral.collateralBalance)`

 (Couldn't comment that line directly.)

Reply:
fixed with wrapping `deleverageCollateralToken`